tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+    "compileOnSave": true,
+    "compilerOptions": {
+        /* es2015 requires moduleResolution and module to be set, 
+        see https://github.com/Microsoft/TypeScript/issues/8189 */
+        "target": "es2015",
```

Comment:
@sindresorhus Might have missed this, but didn't we want to target `ES5`?

Reply:
@SamVerschueren Can't: https://github.com/sindresorhus/is/pull/28#issuecomment-341303651